### DISCUSSION TITLE
[Email Templates] email subject i18n

### DISCUSSION THREAD

**Discussion by naskio** [directus/directus#12249] [Archive]
Hello
Is there a way to translate the email subject (for example the string: "You've been invited")
Located here: https://github.com/directus/directus/blob/93122a0131/api/src/services/users.ts
Thanks

**Comment by azrikahar** [comment]
Currently there isn't a way to translate the subject yet, but PRs such as #7412 and #5446 does help to close the gap towards supporting this. There is also an existing discussion #8239 on this very topic so feel free to chime in over there so that we can converge similar discussions better 👍
